src/daemon/config.rs

```diff
@@ -724,8 +727,10 @@ impl Config {
 
     pub fn read_config(file: &str) -> Result<Self, ConfigError> {
         let mut v = Vec::new();
-        let mut f = File::open(file)?;
-        f.read_to_end(&mut v)?;
+        let mut f =
+            File::open(file).map_err(|e| KrillIoError::new(format!("Could not read config file '{}'", file), e))?;
+        f.read_to_end(&mut v)
+            .map_err(|e| KrillIoError::new(format!("Could not read config file '{}'", file), e))?;
```

Comment:
Same comment as above, will it be possible to distinguish the open and read cases from `e` alone?

Reply:
changed first 'read' to 'open'